test/e2e/main_test.go

```diff
@@ -39,24 +39,29 @@ func TestMain(m *testing.M) {
 		exitCode int
 	)
 
+	if err = e2eframework.SplitFlags(); err != nil {
+		log.Error("Error parsing regular test flags:", err)
+		os.Exit(1)
+	}
+
 	if framework, err = e2eframework.NewFromFlags(); err != nil {
-		log.Printf("failed to setup framework: %v\n", err)
+		log.Errorf("failed to setup framework: %v\n", err)
```

Comment:
new line is not needed here and below
```suggestion
		log.Errorf("failed to setup framework: %v", err)
```